tokio/src/runtime/coop.rs

```diff
@@ -119,6 +119,17 @@ cfg_rt_multi_thread! {
     pub(crate) fn set(budget: Budget) {
         let _ = context::budget(|cell| cell.set(budget));
     }
+
+    /// Consume one budget.
```

Comment:
nit, take it or leave it: I might say something like
```suggestion
    /// Consume one unit of progress from the current task's budget.
```
or something?